api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java

```diff
@@ -874,6 +890,39 @@ public void updateVerifiedInstitutionalAffiliation_removeForbidden() {
     profileController.updateVerifiedInstitutionalAffiliation(dbUser.getUserId(), null);
   }
 
+  @Test
+  public void adminUpdateProfile_disable() {
```

Comment:
new tests

AuthDomainController.updateUserDisabledStatus() is the standalone version of the API

Reply:
Send these two APIs to the Thunderdome then.